stingray/crossspectrum.py

```diff
@@ -1404,6 +1404,76 @@ def from_lightcurve(
             gti=gti,
         )
 
+    @staticmethod
+    def from_stingray_timeseries(
+        lc1,
```

Comment:
same in powerspectrum.py, 
the names of the variables (lc1 and lc2) can be confusing, but I don't know if it's worth changing them in ts1 and ts2